monitor/nuro-data-collector/src/main/java/exception/MetricNotAvailableException.java

```diff
@@ -1,6 +1,6 @@
 /**
- * Copyright 2014 SeaClouds
- * Contact: Dionysis Athanasopoulos <dionysiscsuoi@gmail.com>
+ * Copyright 2015 SeaCloudsEU
+ * Contact: Michele Guerriero <michele.guerriero@mail.polimi.it>
```

Comment:
I noticed that there were a kind of licensing process in SeaClouds and I though it was necessary to add this header. IN example if I'm not wrong the SLA module adopt the same way of licensing (by having an header file and a maven plugin setting this header on top of each class). Also other component classes are licensed, here's just an example https://github.com/SeaCloudsEU/SeaCloudsPlatform/blob/master/dashboard/src/main/java/eu/seaclouds/platform/dashboard/config/BasicEndpointFactory.java .Also here if this is wrong I can delete the header and the plugin. But maybe I just used the wrong ay of adding license?